examples/superdense_coding.py

```diff
@@ -0,0 +1,86 @@
+"""Superdense Coding.
+Superdense Coding is a method to transmit two classical bits of information
+by sending only one qubit of information. This is accomplished by
+pre-sharing an entangled state between the sender and the receiver. This
+entangled state allows the receiver of the one qubit of information to
+decode the two classical bits that were originally encoded by the sender.
+
+In the following example output, a sender sets qubit 0 (i.e., q0) to 0 and
+qubit 1 (i.e., q1) to 1. By sharing one qubit of information (i.e., q2),
+the receiver is able to decode the original 0 and 1 in qubits 3 and 4,
+respectively, when qubits 3 and 4 are measured. This is only possible given
+that an entangled state is pre-shared between the sender and receiver.
+
+In the actual program below, qubit 0 and qubit 1 are randomly set to
+either 1 or 0.
+
+=== REFERENCE ===
+https://en.m.wikipedia.org/wiki/Superdense_coding
+
+=== EXAMPLE OUTPUT ===
+Circuit:
+0: ---------------M-------@-------------------
+                          |
+1: ---X-----------M---@---|-------------------
+                      |   |
+2: -------H---@-------X---@---x---------------
+              |               |
+3: -----------|---------------x---@---H---M---
+              |                   |
+4: -----------X-------------------X-------M---
+
+Results:
+0=0
+1=1
+3=0
+4=1
+
+"""
+
+import random
+import cirq
+
+
+def make_superdense_circuit():
+    circuit = cirq.Circuit()
+    (q0, q1, q2, q3, q4) = cirq.LineQubit.range(5)
+
+    # Randomly sets q0 and q1 to either 0 or 1
+    if random.randint(0, 1):
+        circuit.append([cirq.X(q0)]) # coverage: ignore
```

Comment:
You don't need the square brackets when appending single items.

Reply:
Thanks very much for the heads-up here. That line has now been removed given the change to how these are randomized, though very good to know for the future!